docs/source/index.rst

```diff
@@ -1,25 +1,41 @@
 Piccolo Admin
 =============
 
-`Piccolo Admin <https://github.com/piccolo-orm/piccolo_admin>`_ provides a
-simple yet powerful admin interface on top of Piccolo tables - allowing you to
-easily add / edit / filter your data.
+`Piccolo Admin <https://github.com/piccolo-orm/piccolo_admin>`_ is a powerful
+admin interface / content management system for Python.
 
 .. image:: https://raw.githubusercontent.com/piccolo-orm/piccolo_admin/master/docs/images/screenshot.png
 
 It was created at a design agency to serve the needs of customers who demand a
-high quality, beautiful admin interface for their websites. Our goal is to
-create a world class admin interface, akin to Wordpress or Django.
+high quality, beautiful admin interface for their websites. It's a modern
+alternative to tools like Wordpress and Django Admin.
 
-It's built using the latest technologies, with Vue JS on the front end, and a
-modern REST backend.
+It's built using the latest technologies, with Vue.js on the front end, and a
+powerful REST backend.
 
-`Try it online <https://demo1.piccolo-orm.com/#/login?nextURL=%2F>`_ (username:
-piccolo, password: piccolo123).
+Some of it's standout features:
+
+* Powerful data filtering
+* Builtin security
+* Media support, both locally and in S3 compatible services
+* Dark mode support
+* CSV exports
+* Easily create custom forms
+* Works on mobile and desktop
+* Use standalone, or integrate it easily with ASGI apps like FastAPI, and Starlette
+* Multilingual out of box
```

Comment:
I think you should add `bulk data operations` and `simple table and column configuration` (how data is displayed -`TableConfig`), because those are two really nice features

Reply:
Good idea - I'll add those.